zkevm-circuits/src/copy_circuit/dev.rs

```diff
@@ -68,11 +74,28 @@ impl<F: Field> Circuit<F> for CopyCircuit<F> {
             challenge_values.evm_word(),
         )?;
 
+        #[cfg(not(feature = "dual_bytecode"))]
         config.0.bytecode_table.dev_load(
             &mut layouter,
             self.external_data.bytecodes.values(),
             &challenge_values,
         )?;
+
+        // when enable feature "dual_bytecode", get two sets of bytecodes here.
+        #[cfg(feature = "dual_bytecode")]
+        {
+            let (first_bytecodes, second_bytecodes) =
+                Block::split_two_bytecodes(&self.external_data.bytecodes, &self.bytecode_map);
```

Comment:
again.. not 2 bytecodes, but 2 set of or 2 group of bytecodes.

Reply:
updated in [e15a29e](https://github.com/scroll-tech/zkevm-circuits/pull/1368/commits/e15a29e0af5909592913efb3dd1a61589baccf01)